### DISCUSSION TITLE
[Feature]: Dedicated section for videos (sorting for videos)

### DISCUSSION THREAD

**Discussion by samip5** [immich-app/immich#1624] [Feature Request]
### Feature detail

It would be nice to be able to sort photos and videos separately.

### Platform

Web

**Comment by bo0tzz** [comment]
Is this covered by the new media type filter in the search? If not, can you elaborate on what you mean exactly?

**Reply by samip5** [reply]
It's been ages since I have tried the app, but the general want was a section dedicated to videos, that is not search. It could be timeline but with videos alone, and the search last time I used it, did not feel good to use with videos as one was expecting more of a timeline experience.

(Seems that I had forgotten to answer)
